plugins/net_plugin/net_plugin.cpp

```diff
@@ -1002,22 +1056,17 @@ namespace eosio {
 
 
    template<typename Function>
-   void net_plugin_impl::for_each_connection( Function&& f ) const {
+   void connections_manager::for_each_connection( Function&& f ) const {
```

Comment:
I'd have these small templates inline in the class. And also `std::forward<Function>(f)` at the call location.

Reply:
`for_each_block_connection` can't be include because it uses `connection` `is_transactions_only_connection`. So moved them both out to be consistent.